src/abstractdataframe/abstractdataframe.jl

```diff
@@ -1534,16 +1548,30 @@ function fillcombinations(df::AbstractDataFrame, indexcols;
     return out_df
 end
 
+"""
+MergeDuplicates = Union{Nothing,Function}
+
+Wherever the `mergeduplicates` keyword argument is available it is either `nothing` or
+a `Function` that will be executed to combine duplicated columns (when `makeunique=false`)
+"""
+MergeDuplicates = Union{Nothing,Function}
```

Comment:
I am OK to add this definition, but then its docstring should be more precise I think.